test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -61,6 +61,16 @@ namespace integration_framework {
 
     std::shared_ptr<TestIrohad> instance_;
 
+    std::string getRedisHost() {
+      auto redis_host = std::getenv("IROHA_REDIS_HOST");
```

Comment:
The getter shouldn't perform some logic inside. Better to move it into initialization and fetch after.

Reply:
I suggest to rename redis and postgres functions to `getXOrDefault` and introduce arguments with default values.

Reply:
This commit (aca7373a91, orig: c460c2a) actually from #869 actually will be removed after rebasing. So quite out of the pr scope